config/initializers/datadog-tracer.rb

```diff
@@ -4,5 +4,9 @@
   Datadog.configure do |c|
     c.env = Rails.env
     c.service = 'moth.social'
+    c.tracing.instrument :httprb, service_name: 'moth.social'
+    c.tracing.instrument :faraday, service_name: 'moth.social'
+    c.tracing.instrument 'active_support-cache'.to_sym, service_name: 'moth.social'
+    c.tracing.instrument :http, service_name: 'moth.social'
```

Comment:
i remember trying to do this in prod some time ago and it was failing. can you test this in staging before pushing to prod, please? that's because if the initializer errors out, the service will fail to start and the service will be down until we revert

Reply:
yes, for sure will test in staging.

Reply:
I think the errors I saw were that the method I was calling on the datadog object didn't exist or something like that. can't remember exactly